app/code/Magento/GiftMessage/Model/Plugin/OrderItemGet.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\GiftMessage\Model\Plugin;
+
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Sales\Api\Data\OrderItemInterface;
+use Magento\Sales\Api\OrderItemRepositoryInterface;
+use Magento\GiftMessage\Api\OrderItemRepositoryInterface as GiftMessageItemRepositoryInterface;
+use Magento\Sales\Api\Data\OrderItemExtensionFactory;
+
+/**
+ * Plugin for adding gift message to order item
+ */
+class OrderItemGet
+{
+    /** @var OrderItemExtensionFactory */
```

Comment:
Please use the following format of phpdoc for consistency

    /**
     * @var OrderItemExtensionFactory 
     */